src/api-service/__app__/onefuzzlib/orm.py

```diff
@@ -281,18 +286,44 @@ def get_keys(self) -> Tuple[KEY, KEY]:
 
         return (partition_key, row_key)
 
+    @classmethod
+    def hide_secrets(
+        cls,
+        model: BaseModel,
+        hider: Callable[["SecretData"], None],
+        visited: Set[int] = set(),
+    ) -> None:
+        model_type = type(model)
+        if Type_has_secrets.get(model_type, True):
+            for field in model.__fields__:
```

Comment:
If a field is List[Field], which is common, this doesn't iterate over those nested fields.

Reply:
Probably needs to support isinstance of `list` and the values of `dict`